app/models/user.rb

```diff
@@ -1,15 +1,12 @@
 class User < ActiveRecord::Base
   include TokenAuthenticatable
   # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable and :omniauthable
+  # :token_authenticatable, :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable,
          :recoverable, :rememberable,
-         :trackable, :validatable,
-         :token_authenticatable
+         :trackable, :validatable
   has_many :orders
   validates :name, presence: true
-  validates :email, presence: true, uniqueness: true
-  validates_format_of :email, with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\Z/i
```

Comment:
Devise is taking care of that with the `:validatable` flag in User Model

`curl -X POST http://localhost:3000/v1/users --data "name=John&email=1@usertmag&password=password&p1assword_confirmation=password"`
returns:
`{"message":["Email is invalid"]}`